Merchant/Core/Services/SecureAccessService.swift

```diff
@@ -19,9 +19,7 @@ class SecureAccessService: PinDelegate {
         self.successHandler = successHandler
         self.errorHandler = errorHandler
         
-        guard let window = UIApplication.shared.keyWindow
-            , let currentViewController = window.rootViewController else {
-                // Manage the error here
+        guard let window = UIApplication.shared.keyWindow, let currentViewController = window.rootViewController else {
```

Comment:
No change required.

Reply:
This is part of the commit which removed unnecessary comments I believe. Here was a comment saying `Manage the error here`. If there is an error to manage it should be managed and not just a comment saying that...

Reply:
You can change the comment by `TODO :`. However I was not talking about the comment but the code style.

Reply:
Well, you haven't really said that 😅Talking about `TODO:` you are completely right and I've been thinking about changing some of the comments to `TODO`. The reason I didn't is because I haven't found any comments like that and because if there is anything "TO-DO" I think it should be simply done/scheduled and not pushed into a production.

Talking about style of the code. Again, I don't think you guys have here any style-guide that I could follow. If I remember correctly I also think there can be found both alternatives in the project.